Opensourcecommunity/Contributing-to-the-code.md

```diff
@@ -5,7 +5,7 @@ Thank you for contributing to the Open Smart Grid Platform. Please keep the foll
 ### Guidelines
 
 Before code is merged it needs to comply with a number of guidelines:
-1. Code should be as complete as possible (No placeholders, TODO's or FIXME's)
+1. Code should be as complete as possible (preferably no placeholders, TODO's or FIXME's)
```

Comment:
What do you guys think of this change? I still see TODO's in pull-requests sometimes. This reflects the current way of working.